docker-compose.yml

```diff
@@ -74,8 +70,6 @@ services:
       - "oar.wsgi"
     ports:
       - "8081:8081"
-    networks:
```

Comment:
@VadimKovalenkoSNF @vladsha-dev Did you test Dedupe Hub without networks configuration locally?

Reply:
I haven't tested changes related to OSDEV-1002 yet.